pkg/task/backup.go

```diff
@@ -65,6 +67,7 @@ type BackupConfig struct {
 	GCTTL            int64         `json:"gc-ttl" toml:"gc-ttl"`
 	RemoveSchedulers bool          `json:"remove-schedulers" toml:"remove-schedulers"`
 	IgnoreStats      bool          `json:"ignore-stats" toml:"ignore-stats"`
+	UseBackupMetaV2  bool          `json:"use-backupmeta-v2"`
```

Comment:
this should better be a flag accepting an integer like `--backupmeta-version=2`, in case we need a v3 in the future.

Reply:
yes, that sounds better, but it's a hidden flag, let's change it in meta v3.